docs/source/integrations/burrow_example.rst

```diff
@@ -150,85 +150,120 @@ Note the last line of the send function that emits a Sent event which gets recor
 	emit Sent(msg.sender, receiver, amount);
 
 
-Case 2. Querying Iroha state
+Case 2. Interacting with Iroha state
 ----------------------------
 
 Earlier we looked at an example of a contract that didn’t interact with Iroha state.
 However, in most real life applications one could imagine running on top of Iroha blockchain (like custom business logic in transaction processing or charging transaction fees etc.) being able to interact with Iroha state is indispensable.
 In this section we will consider an example of how one can query balances of Iroha accounts (provided the query creator has respective permissions) from inside an EVM smart contract.
 
 
-The code of the contract is presented on the diagram below:
+Here is a sample code of contact to do so:
 
 .. code-block:: solidity
 
-	contract QueryIroha {
-	    address public serviceContractAddress;
-
-	    // Initializing service contract address in constructor
-	    constructor() public {
-	        serviceContractAddress = 0xA6Abc17819738299B3B2c1CE46d55c74f04E290C;
-	    }
-
-	    // Queries the balance in _asset of an Iroha _account
-	    function queryBalance(string memory _account, string memory _asset) public
-	                    returns (bytes memory result) {
-	        bytes memory payload = abi.encodeWithSignature(
-	            "getAssetBalance(string,string)",
-	            _account,
-	            _asset);
-	        (bool success, bytes memory ret) =
-	            address(serviceContractAddress).delegatecall(payload);
-	        require(success, "Error calling service contract function");
-	        result = ret;
-	    }
+	contract Iroha {
+		address public serviceContractAddress;
```

Comment:
If the code is somewhere in code I suggest to include its directly from file instead of copy-paste.

Reply:
I have given the link to file at the end as well, I thought providing a sample code here as well would be good.

Reply:
Currently there is no fixed source of code in the repository, since it is not merged yet.